std/net/curl.d

```diff
@@ -2314,8 +2313,17 @@ private bool decodeLineInto(Terminator, Char = char)(ref const(ubyte)[] basesrc,
   *     (in char[] key, in char[] value) { writeln(key ~ ": " ~ value); };
   * http.onReceive = (ubyte[] data) { /+ drop +/ return data.length; };
   * http.perform();
+  * ---
+  *
+  */
+
+/**
+  * Put with data senders:
+  *
+  * ---
+  * import std.net.curl, std.stdio;
   *
-  * // Put with data senders
+  * auto http = HTTP();
```

Comment:
**Probably requires a working host.**
```
auto http = HTTP("dlang.org");
```
This examples probably never worked, since it requires a working host that provides response. But I'm not sure.